src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.h

```diff
@@ -34,6 +34,7 @@
 #include <string>                 // for string
 #include <sys/types.h>            // for key_t
 #include <vector>                 // for vector
+#include "spoofing_detector.h"
```

Comment:
ah. Here is the declaration. This needs to be included in **rtklib_pvt** (probably in the header, since you should make the detector a member), or else you should move the construction of the `SpoofingDetector` to this class.